doc/lsst.meas.extensions.shapeHSM/index.rst

```diff
@@ -7,6 +7,20 @@ lsst.meas.extensions.shapeHSM
 #############################
 
 The ``lsst.meas.extensions.shapeHSM`` module provides algorithms for HSM shape measurement.
+The algorithm was initially described in `Hirata & Seljak (2003) <https://ui.adsabs.harvard.edu/abs/2003MNRAS.343..459H/abstract>`_, and was modified later in `Mandelbaum et al. (2005) <https://ui.adsabs.harvard.edu/abs/2005MNRAS.361.1287M/abstract>`_.
+HSM is named after the primary authors: i) Hirata, Christopher, ii) Seljak, Uros and iii) Mandelbaum, Rachel.
```

Comment:
I think it scans better if the names are just written out: people can figure out the abbreviation pretty easily from that.

```suggestion
HSM is named after the primary authors: Christopher Hirata, Uros Seljak, and Rachel Mandelbaum.
```